src/components-examples/material/checkbox/checkbox-overview/checkbox-overview-example.html

```diff
@@ -1 +1,25 @@
-<mat-checkbox>Check me!</mat-checkbox>
+<section class="example-section">
+  <mat-checkbox class="example-margin">Check me!</mat-checkbox>
+  <mat-checkbox class="example-margin" [disabled]=true>Disabled</mat-checkbox>
+</section>
+
+<section class="example-section">
+  <span class="example-list-section">
+    <mat-checkbox class="example-margin"
+                  [(ngModel)]="task.completed"
+                  [checked]="allComplete()"
+                  [indeterminate]="someComplete()"
+                  (change)="setAll($event.checked)">
```

Comment:
Since we're using `ngModel` above, it might be better to use the `ngModelChange` event here for consistency.

Reply:
This doesn't quite work, since the `ngModel` was task.completed and it's not looking at the right object. Instead, I removed `ngModel` and use `change`. Does that seem ok?